blocks/ticket/ticket.ts

```diff
@@ -150,10 +169,22 @@ export class Ticket {
                         const mapping =
                           ticketingWorkflowVarMapping[v.fieldType];
 
-                        const filteredVars = allWorkflowVars.filter((v) =>
-                          mapping ? mapping.includes(v.type) : true
+                        let filteredVars = allWorkflowVars.filter(
+                          (workflowVar) =>
+                            !mapping ||
+                            mapping?.includes(workflowVar.type) ||
+                            // for single select field, we allow COMP variable
+                            (v.fieldType === "SEL" &&
+                              /^COMP\d+/.test(workflowVar.label))
```

Comment:
COMP var has type 'PARA', but product wants single select field to only have TXT and COMP var. Workaround here to check the label since COMP label is not modifiable afaik